### DISCUSSION TITLE
Question on embedding CC links

### DISCUSSION THREAD

**Discussion by geniza** [CollectionBuilder/collectionbuilder.github.io#56] [Q&A] [answered]
Hey there, thanks again for the guidance on the image display issue. I've continued building my collection since then but have encountered another issue I can't seem to trace/troubleshoot. On each of the object pages I have included a "Rights Statement" and a "Citation(s)" field. Oddly enough, the "Rights Statement" is not appearing as a hyperlink for each object, however, the "Citation(s)" field is. How do I ensure that the opposite is the case so that the CC statements appear as hyperlinks and the citations appear as plain text? (for example: [https://geniza.github.io/paleographicalglossary/item.html?id=ccg0006] )
![Screen Shot 2022-06-23 at 12 07 07 PM](https://user-images.githubusercontent.com/106448975/175345108-b1025693-3bc5-4f30-91f2-2d16ac801d66.png)

**Comment by dcnb** [comment, marked as answer]
Hi @geniza! 

Your config-metadata.csv is to blame! 

The last two lines should read like this: 

rightsstatement,Rights Statement,,true
citation, Citation(s),,

Right now, the true value--which turns on an external link--is listed for the citation field. If you switch that around, everything should work. 

Cheers!
@dcnb

**Reply by geniza** [reply]
Great, thanks so much @dcnb! That change resolved everything :)
